zebra-rpc/src/methods/tests/vectors.rs

```diff
@@ -305,3 +305,129 @@ async fn rpc_getrawtransaction() {
     let rpc_tx_queue_task_result = rpc_tx_queue_task_handle.now_or_never();
     assert!(matches!(rpc_tx_queue_task_result, None));
 }
+
+#[tokio::test]
+async fn rpc_getaddresstxids_invalid_arguments() {
+    zebra_test::init();
+
+    let mut mempool: MockService<_, _, _, BoxError> = MockService::build().for_unit_tests();
+
+    // Create a continuous chain of mainnet blocks from genesis
+    let blocks: Vec<Arc<Block>> = zebra_test::vectors::CONTINUOUS_MAINNET_BLOCKS
+        .iter()
+        .map(|(_height, block_bytes)| block_bytes.zcash_deserialize_into().unwrap())
+        .collect();
+
+    // Create a populated state service
+    let (_state, read_state, latest_chain_tip, _chain_tip_change) =
+        zebra_state::populated_state(blocks.clone(), Mainnet).await;
+
+    let rpc = RpcImpl::new(
+        "RPC test",
+        Buffer::new(mempool.clone(), 1),
+        Buffer::new(read_state.clone(), 1),
+        latest_chain_tip,
+        Mainnet,
+    );
+
+    // call the method with an invalid address string
+    let address = "11111111".to_string();
+    let addresses = vec![address.clone()];
+    let start: u32 = 1;
+    let end: u32 = 2;
+    let error = rpc
```

Comment:
After PR #4015, the queue task gets returned here, along with the RPC server.

Here's the fix that the tests need:
https://github.com/ZcashFoundation/zebra/pull/4015/commits/2890b85b0f92c895a608f85a3dfc270584888210

Reply:
Fixed at https://github.com/ZcashFoundation/zebra/pull/4062/commits/cc376590fa924c362dd92c9be8ba7dfed71d2bf5